getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Quantum Teleportation Protocol \n",
+    "\n",
+    "Teleportation is achieved in four steps:\n",
+    "\n",
+    "- **Initial state:** Alice starts with a quantum state $|\\psi\\rangle$ of a single qubit. Alice and Bob also share a quantum state of two qubits, $\\frac{|00\\rangle+|11\\rangle}{\\sqrt 2}$\n",
```

Comment:
Worth reminding the reader that this two-qubit state is a "Bell pair".

Reply:
Updated